src/components/App/App.css

```diff
@@ -4,7 +4,6 @@
   text-align: center;
   justify-content: flex-end;
   align-items: center;
-  height: 100%;
```

Comment:
This height isn't necessary, it is helpful not to have it when components will overflow